farm_import_csv kernel tests failing

The kernel tests for the `farm_import_csv` module started failing in our nightly automated test runs via GitHub Actions on November 6th (~2 weeks ago).

```
ParaTest v6.11.1 upon PHPUnit 9.6.21 by Sebastian Bergmann and contributors.

Processes:     4
Runtime:       PHP 8.2.25
Configuration: /opt/drupal/phpunit.xml

................................................................. 65 / 95 ( 68%)
..................E.EE.EE.....                                    95 / 95 (100%)

Time: 03:44.054, Memory: 12.00 MB

There were 5 errors:

1) Drupal\Tests\farm_import_csv\Kernel\AssetCsvImportTest::testAssetCsvImport
Drupal\Core\Database\DatabaseExceptionWrapper: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'farm.test44644271migrate_tools_sync_source_ids' doesn't exist: DELETE FROM "test44644271migrate_tools_sync_source_ids" 
WHERE "migration_id" = :db_condition_placeholder_0; Array
(
    [:db_condition_placeholder_0] => csv_asset:equipment
)


/opt/drupal/web/core/modules/mysql/src/Driver/Database/mysql/ExceptionHandler.php:56
/opt/drupal/web/core/lib/Drupal/Core/Database/Query/Delete.php:63
/opt/drupal/web/modules/migrate_tools/src/MigrateTools.php:87
/opt/drupal/web/modules/migrate_tools/src/EventSubscriber/MigrationImportSync.php:125
/opt/drupal/web/core/lib/Drupal/Component/EventDispatcher/ContainerAwareEventDispatcher.php:111
/opt/drupal/web/core/modules/migrate/src/MigrateExecutable.php:307
/opt/drupal/web/core/modules/migrate/tests/src/Kernel/MigrateTestBase.php:179
/opt/drupal/web/profiles/farm/modules/core/import/modules/csv/tests/src/Kernel/CsvImportTestBase.php:123
/opt/drupal/web/profiles/farm/modules/core/import/modules/csv/tests/src/Kernel/AssetCsvImportTest.php:53
/opt/drupal/vendor/phpunit/phpunit/src/Framework/TestResult.php:729

2) Drupal\Tests\farm_import_csv\Kernel\ConfigCsvImportTest::testConfigCsvImport
Drupal\Core\Database\DatabaseExceptionWrapper: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'farm.test32156576migrate_tools_sync_source_ids' doesn't exist: DELETE FROM "test32156576migrate_tools_sync_source_ids" 
WHERE "migration_id" = :db_condition_placeholder_0; Array
(
    [:db_condition_placeholder_0] => egg_harvest
)


/opt/drupal/web/core/modules/mysql/src/Driver/Database/mysql/ExceptionHandler.php:56
/opt/drupal/web/core/lib/Drupal/Core/Database/Query/Delete.php:63
/opt/drupal/web/modules/migrate_tools/src/MigrateTools.php:87
/opt/drupal/web/modules/migrate_tools/src/EventSubscriber/MigrationImportSync.php:125
/opt/drupal/web/core/lib/Drupal/Component/EventDispatcher/ContainerAwareEventDispatcher.php:111
/opt/drupal/web/core/modules/migrate/src/MigrateExecutable.php:307
/opt/drupal/web/core/modules/migrate/tests/src/Kernel/MigrateTestBase.php:179
/opt/drupal/web/profiles/farm/modules/core/import/modules/csv/tests/src/Kernel/CsvImportTestBase.php:123
/opt/drupal/web/profiles/farm/modules/core/import/modules/csv/tests/src/Kernel/ConfigCsvImportTest.php:37
/opt/drupal/vendor/phpunit/phpunit/src/Framework/TestResult.php:729

3) Drupal\Tests\farm_import_csv\Kernel\CsvImportDatabaseTest::testDatabaseTable
Drupal\Core\Database\DatabaseExceptionWrapper: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'farm.test48659854migrate_tools_sync_source_ids' doesn't exist: DELETE FROM "test48659854migrate_tools_sync_source_ids" 
WHERE "migration_id" = :db_condition_placeholder_0; Array
(
    [:db_condition_placeholder_0] => csv_taxonomy_term:animal_type
)


/opt/drupal/web/core/modules/mysql/src/Driver/Database/mysql/ExceptionHandler.php:56
/opt/drupal/web/core/lib/Drupal/Core/Database/Query/Delete.php:63
/opt/drupal/web/modules/migrate_tools/src/MigrateTools.php:87
/opt/drupal/web/modules/migrate_tools/src/EventSubscriber/MigrationImportSync.php:125
/opt/drupal/web/core/lib/Drupal/Component/EventDispatcher/ContainerAwareEventDispatcher.php:111
/opt/drupal/web/core/modules/migrate/src/MigrateExecutable.php:307
/opt/drupal/web/core/modules/migrate/tests/src/Kernel/MigrateTestBase.php:179
/opt/drupal/web/profiles/farm/modules/core/import/modules/csv/tests/src/Kernel/CsvImportTestBase.php:123
/opt/drupal/web/profiles/farm/modules/core/import/modules/csv/tests/src/Kernel/CsvImportDatabaseTest.php:43
/opt/drupal/vendor/phpunit/phpunit/src/Framework/TestResult.php:729

4) Drupal\Tests\farm_import_csv\Kernel\LogCsvImportTest::testLogCsvImport
Drupal\Core\Database\DatabaseExceptionWrapper: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'farm.test44595028migrate_tools_sync_source_ids' doesn't exist: DELETE FROM "test44595028migrate_tools_sync_source_ids" 
WHERE "migration_id" = :db_condition_placeholder_0; Array
(
    [:db_condition_placeholder_0] => csv_log:harvest
)


/opt/drupal/web/core/modules/mysql/src/Driver/Database/mysql/ExceptionHandler.php:56
/opt/drupal/web/core/lib/Drupal/Core/Database/Query/Delete.php:63
/opt/drupal/web/modules/migrate_tools/src/MigrateTools.php:87
/opt/drupal/web/modules/migrate_tools/src/EventSubscriber/MigrationImportSync.php:125
/opt/drupal/web/core/lib/Drupal/Component/EventDispatcher/ContainerAwareEventDispatcher.php:111
/opt/drupal/web/core/modules/migrate/src/MigrateExecutable.php:307
/opt/drupal/web/core/modules/migrate/tests/src/Kernel/MigrateTestBase.php:179
/opt/drupal/web/profiles/farm/modules/core/import/modules/csv/tests/src/Kernel/CsvImportTestBase.php:123
/opt/drupal/web/profiles/farm/modules/core/import/modules/csv/tests/src/Kernel/LogCsvImportTest.php:81
/opt/drupal/vendor/phpunit/phpunit/src/Framework/TestResult.php:729

5) Drupal\Tests\farm_import_csv\Kernel\TermCsvImportTest::testTermCsvImport
Drupal\Core\Database\DatabaseExceptionWrapper: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'farm.test70384369migrate_tools_sync_source_ids' doesn't exist: DELETE FROM "test70384369migrate_tools_sync_source_ids" 
WHERE "migration_id" = :db_condition_placeholder_0; Array
(
    [:db_condition_placeholder_0] => csv_taxonomy_term:animal_type
)


/opt/drupal/web/core/modules/mysql/src/Driver/Database/mysql/ExceptionHandler.php:56
/opt/drupal/web/core/lib/Drupal/Core/Database/Query/Delete.php:63
/opt/drupal/web/modules/migrate_tools/src/MigrateTools.php:87
/opt/drupal/web/modules/migrate_tools/src/EventSubscriber/MigrationImportSync.php:125
/opt/drupal/web/core/lib/Drupal/Component/EventDispatcher/ContainerAwareEventDispatcher.php:111
/opt/drupal/web/core/modules/migrate/src/MigrateExecutable.php:307
/opt/drupal/web/core/modules/migrate/tests/src/Kernel/MigrateTestBase.php:179
/opt/drupal/web/profiles/farm/modules/core/import/modules/csv/tests/src/Kernel/CsvImportTestBase.php:123
/opt/drupal/web/profiles/farm/modules/core/import/modules/csv/tests/src/Kernel/TermCsvImportTest.php:39
/opt/drupal/vendor/phpunit/phpunit/src/Framework/TestResult.php:729

FAILURES!
Tests: 95, Assertions: 3015, Errors: 5.
Error: Process completed with exit code 2.
```

--- a/modules/core/import/modules/csv/tests/src/Kernel/CsvImportTestBase.php
+++ b/modules/core/import/modules/csv/tests/src/Kernel/CsvImportTestBase.php
@@ -74,6 +74,7 @@ class CsvImportTestBase extends MigrateTestBase {
     $this->installEntitySchema('taxonomy_term');
     $this->installEntitySchema('user');
     $this->installConfig(['farm_format', 'farm_entity_views', 'farm_quantity_standard', 'farm_import_csv']);
+    $this->installSchema('migrate_tools', ['migrate_tools_sync_source_ids']);
     $this->installSchema('farm_import_csv', ['farm_import_csv_entity']);
 
     // Run tests as the user 1 to avoid permissions issues.

